lib/Excel/Writer/XLSX.pm

```diff
@@ -448,15 +448,19 @@ The properties that can be set are:
     title
```

Comment:
> I was disappointed to not see "created" in the list initially and went
about looking for other solutions before finally digging into the source
and realizing the it was actually supported!

So this is a feature request then?